examples/poc/README.md

```diff
@@ -0,0 +1,63 @@
+# Envoy Ext Proc Gateway with LoRA Integration
+
+This project sets up an Envoy gateway to handle gRPC calls with integration of LoRA (Low-Rank Adaptation). The configuration aims to manage gRPC traffic through Envoy's external processing and custom routing based on headers and load balancing rules. The setup includes Kubernetes services and deployments for both the gRPC server and the vllm-lora application.
+
+## Requirements
+- A vLLM based deployment (using the custom image provided below), with LoRA Adapters
+- Kubernetes cluster
+- Envoy Gateway v1.1 installed on your cluster: https://gateway.envoyproxy.io/v1.1/tasks/quickstart/
+- `kubectl` command-line tool
+- Go (for local development)
+
+## vLLM
+***This PoC uses a modified vLLM fork, the public image of the fork is here: `ghcr.io/tomatillo-and-multiverse/vllm:demo`***
```

Comment:
Is there a reason to use a modified fork?

Reply:
Document why the fork.

Reply:
And specifically link to the commits/commit-range/branch/PR that added them.

Reply:
Done. I linked to the fork, there is not currently a PR for the fork